### DISCUSSION TITLE
Actions Docker bug

### DISCUSSION THREAD

**Discussion by nihaals** [community/community#24983] [Actions]
<h1><a href="https://github.com/nihaals/actions-docker-arch-mwe" rel="noopener nofollow ugc">Minimum Working Example</a></h1>
<p>This issue has been introduced over the last few days. I’m not sure if it’s an Arch issue, but considering it builds fine locally this seems to be an Actions issue. There’s more info on what I’ve already tried and where the issue is coming from on a <a href="https://stackoverflow.com/questions/66088277/unexpected-bash-readable-test-result-with-github-actions" rel="noopener nofollow ugc">StackOverflow question</a>.</p>

**Comment by yan12125** [comment]
<p>We got the same “(could not find or read directory: /var/lib/pacman/)” error and our workaround may help: <a href="https://github.com/lxqt/lxqt-panel/pull/1562/files" class="inline-onebox" rel="noopener nofollow ugc">Workaround an issue with glibc 2.33 on old Docker engines by yan12125 · Pull Request #1562 · lxqt/lxqt-panel · GitHub</a>.</p>

**Comment by nihaals** [comment, marked as answer]
<p>This helped me find an existing <a href="https://github.com/actions/virtual-environments/issues/2658" rel="noopener nofollow ugc">GitHub Issue</a> which seems to be related. It’s probably best to use that instead and hope there’s an official reply</p>
